R/04-webprocess-obj.R

```diff
@@ -83,6 +97,7 @@ setMethod(f="initialize",signature="webprocess",
             .Object@url <- url
             .Object@email <- email
             .Object@wait <- wait
+            .Object@sleep.time <- sleep.time
             
             # // -- supporting pass through of existing inputs arguments *when* they are applicable.
             old.inputs = inputs(.Object)
```

Comment:
I will take a look. I think the reason I do this is because it supports a pass through w/ modify of only the variables you have called specifically. I *think* it is the thing that makes 
```r
geoknife(stencil, fabric, knife, ...) # where ... can be args to knife
```
work.
but I will check that. This is a pattern I played around with as being the best compromise I could find for that type of behavior (if memory serves...).